include/deal.II/meshworker/simple.h

```diff
@@ -457,6 +460,26 @@ namespace MeshWorker
       template<class DOFINFO>
       void assemble(const DOFINFO &info1,
                     const DOFINFO &info2);
+
+    private:
+      /**
+       * A pointer to the object containing constraints.
+       */
+      SmartPointer<const ConstraintMatrix,MatrixSimple<MatrixType> > constraints;
```

Comment:
I am a bit uncomfortable with introducing yet another pointer to constraints (there is at least one in the base class ResidualSimple). Can't you re-use that one?